include/triton/core/tritonserver.h

```diff
@@ -1692,6 +1692,8 @@ TRITONSERVER_DECLSPEC TRITONSERVER_Error*
 TRITONSERVER_ServerOptionsSetCudaMemoryPoolByteSize(
     TRITONSERVER_ServerOptions* options, int gpu_device, uint64_t size);
 
+/// Deprecated. See TRITONSERVER_ServerOptionsSetCacheConfig instead.
```

Comment:
Remove "Deprecated".

Reply:
I don't mind removing it, but I think it's more clear to the user to have "deprecated" included to stand out more. We do so in some other places as well, ex: https://github.com/triton-inference-server/core/blob/46eb9847a096c1d1a8cfa93c528991de926246f5/include/triton/core/tritonserver.h#L680-L683